docs/reference/cluster/remote-info.asciidoc

```diff
@@ -41,8 +41,9 @@ by the configured remote cluster alias.
 
 [[skip-unavailable]]
 `skip_unavailable`::
-    Whether the remote cluster is skipped in case it is searched through
-    a {ccs} request but none of its nodes are available.
+    Whether the remote cluster is skipped in case it is searched through a {ccs}
+    request but none of its nodes are available. Refer to
```

Comment:
here we mention node availability and I am thinking we should update this too.

Whether the remote cluster is skipped in case it is searched through a ccs request but its nodes return errors or are unavailable ?

Reply:
Thanks. I've updated the definition to cover both an unavailable cluster and ignored errors.